blog/_posts/2016-12-19-team-reactives-@-css-and-js-conf-asia.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Team Reactives @ css and js conf Asia
+layout: post
+created_at: Monday Dec 19 2016
+permalink: blog/2016-12-19-team-reactives-@-css-and-js-conf-asia
+current: blog
+author: Team Reactives
+twitter: @shwetha_ravi3
+categories:
+- student-posts
+---
+
+## Hello RGSoC Blog Readers!
+
+Team Reactives here and we have had the amazing opportunity to attend the JS and CSS conf Asia 2016! Kicking off with CSS conf, the conferences featured the likes of Chris Lilley “the father of SVG (Scalable Vector Graphics)” who talked to us about the evolution of CSS4 colors and author Lea Verou who introduced us to the concept of css variables. 
+
+![Can I Use It](/img/blog/2016/team-reactives-cssconf-CanIUseIt.jpg)
+Chris Liley on CSS4 colors - Source: Team Reactives
```

Comment:
please add the same caption style as you can find in the other blog posts (e.g. `<font color="grey"><small><i>Chris Liley on CSS4 colors - Source: Team Reactives</i></small></font>`)